hadoop-tools/hadoop-aliyun/src/main/java/org/apache/hadoop/fs/aliyun/oss/Constants.java

```diff
@@ -211,4 +211,54 @@ private Constants() {
   public static final String LIST_VERSION = "fs.oss.list.version";
 
   public static final int DEFAULT_LIST_VERSION = 2;
+
+  /**
+   * What is the smallest reasonable seek in bytes such
+   * that we group ranges together during vectored read operation.
+   * Value : {@value}.
+   */
+  public static final String OSS_VECTOR_READS_MIN_SEEK_SIZE =
+      "fs.oss.vectored.read.min.seek.size";
+
+  /**
+   * What is the largest merged read size in bytes such
+   * that we group ranges together during vectored read.
+   * Setting this value to 0 will disable merging of ranges.
+   * Value : {@value}.
+   */
+  public static final String OSS_VECTOR_READS_MAX_MERGED_READ_SIZE =
+      "fs.oss.vectored.read.max.merged.size";
+
+  /**
+   * Default minimum seek in bytes during vectored reads : {@value}.
+   */
+  public static final int DEFAULT_OSS_VECTOR_READS_MIN_SEEK_SIZE = 4896; // 4K
+
+  /**
+   * Default maximum read size in bytes during vectored reads : {@value}.
+   */
+  public static final int DEFAULT_OSS_VECTOR_READS_MAX_MERGED_READ_SIZE = 1253376; //1M
```

Comment:
please pick the correct values from https://github.com/apache/hadoop/pull/6702/files

Reply:
OK